aiida_common_workflows/common/visualization/dissociation.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Utilities to visualize a dissociation curve based on set of distances and energies."""
+
+import typing
+
+import pylab as plt
+
+
+def get_dissociation_plot(
+    distances: typing.List[float],
+    energies: typing.List[float],
+    unit_distance: str = 'Å^3',
+    unit_energy: str = 'eV'
+) -> plt:
+    """Plot the dissociation curve for a given set of distances and energies.
+
+    :param distances: list of cell volumes.
+    :param energies: list of energies.
+    :param unit_distance: unit of volume, default is [Å^3].
+    :param unit_energy: unit of energy, default is [eV].
+    """
```

Comment:
Maybe we add some basic validation to make sure the lists are not empty, of equal length and actually contain floats/integers? Let you decide whether you want to add this in this PR though, since it is an addition

Reply:
Seems like something that fits perfectly in the PR I'm prepping for #185 👍